> Please run `yarn check-types` too, one of the checks is failing.

```TypeScript
@lodestar/beacon-node: $ tsc
@lodestar/beacon-node: test/unit/eth1/utils/eth1Data.test.ts:109:77 - error TS2345: Argument of type 'string[]' is not assignable to parameter of type '("depositRoot" | "depositCount" | keyof Eth1Block)[]'.
@lodestar/beacon-node:   Type 'string' is not assignable to type '"depositRoot" | "depositCount" | keyof Eth1Block'.
@lodestar/beacon-node: 109         const eth1DatasPartial = eth1Datas.map((eth1Data) => pick(eth1Data, Object.keys(expectedEth1Data[0])));
@lodestar/beacon-node:                                                                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@lodestar/beacon-node: Found 1 error in test/unit/eth1/utils/eth1Data.test.ts:109
```

--- a/packages/beacon-node/test/unit/eth1/utils/eth1Data.test.ts
+++ b/packages/beacon-node/test/unit/eth1/utils/eth1Data.test.ts
@@ -274,7 +274,7 @@ function getMockDeposit({blockNumber, index}: {blockNumber: number; index: numbe
   };
 }
 
-function pick<T, K extends keyof T>(obj: T, keys: K[] | string[]): Pick<T, K> {
+function pick<T, K extends keyof T>(obj: T, keys: string[]): Pick<T, K> {
   return (keys as K[]).reduce(
     (acc, key) => {
       if (obj && obj.hasOwnProperty(key)) {

--- a/packages/beacon-node/test/unit/eth1/utils/eth1Data.test.ts
+++ b/packages/beacon-node/test/unit/eth1/utils/eth1Data.test.ts
@@ -274,8 +274,8 @@ function getMockDeposit({blockNumber, index}: {blockNumber: number; index: numbe
   };
 }
 
-function pick<T, K extends keyof T>(obj: T, keys: K[]): Pick<T, K> {
-  return keys.reduce(
+function pick<T, K extends keyof T>(obj: T, keys: K[] | string[]): Pick<T, K> {
+  return (keys as K[]).reduce(
     (acc, key) => {
       if (obj && obj.hasOwnProperty(key)) {
         acc[key] = obj[key];

--- a/packages/beacon-node/test/unit/eth1/utils/eth1Data.test.ts
+++ b/packages/beacon-node/test/unit/eth1/utils/eth1Data.test.ts
@@ -106,7 +106,7 @@ describe("eth1 / util / getEth1DataForBlocks", function () {
 
       if (expectedEth1Data) {
         const eth1Datas = await eth1DatasPromise;
-        const eth1DatasPartial = eth1Datas.map((eth1Data) => pick(eth1Data, Object.keys(expectedEth1Data[0])));
+        const eth1DatasPartial = eth1Datas.map(({blockNumber, depositCount}) => ({blockNumber, depositCount}));
         expect(eth1DatasPartial).toEqual(expectedEth1Data);
       } else if (error != null) {
         await expectRejectedWithLodestarError(eth1DatasPromise, error);
@@ -274,14 +274,3 @@ function getMockDeposit({blockNumber, index}: {blockNumber: number; index: numbe
   };
 }
 
-function pick<T, K extends keyof T>(obj: T, keys: string[]): Pick<T, K> {
-  return (keys as K[]).reduce(
-    (acc, key) => {
-      if (obj && obj.hasOwnProperty(key)) {
-        acc[key] = obj[key];
-      }
-      return acc;
-    },
-    {} as Pick<T, K>
-  );
-}

